lib/test.rb

```diff
@@ -1071,6 +1108,7 @@ def devel_only_tap?(formula)
     end
 
     def run
+      @formulae_that_have_been_built = []
```

Comment:
This makes sense to be in `run` so you don't have to recreate the test in order to re-run (though that never happens anyway).